### PR TITLE
Sanity check: calculate entropy of grammar log productions.

### PR DESCRIPTION
**Pull request by CatherineWong**
Submitting this as a pull request rather than merging directly to sanity check:
- the grammar logProductions are the right (unnormalized) logits for each task.
- And then we need to normalize them to calculate entropies.

### CODE REVIEW

**Comment by ellisk42** [comment]
Yes, you are correct that logProductions are the unnormalized logits for each task.
